.github/workflows/generate_api_client.yaml

```diff
@@ -18,14 +18,15 @@ jobs:
     steps:
       - uses: actions/setup-node@v2
       - uses: actions/checkout@v2
-      - run: ./hack/scripts/generate.sh
+      - run: ./scripts/fetch_api.sh
+      - run: ./scripts/generate_golang.sh
       - uses: peter-evans/create-pull-request@v3
         with:
           title: "Update API client"
           token: "${{ env.APP_SERVICES_TOKEN }}"
           commit-message: "chore: generate API client"
           author: app-services-ci <app-services-ci@users.noreply.github.com>
           delete-branch: true
-          reviewers: craicoverflow
+          reviewers: craicoverflow, wtrocki
```

Comment:
I think eventually we can move this out of this action so we are reviewers for all PRs automatically.